ably/types/tokendetails.py

```diff
@@ -35,9 +42,9 @@ def client_id(self):
     @staticmethod
     def from_dict(obj):
         return TokenDetails(
-            id=obj.get("id"),
+            token=obj.get("token"),
             expires=int(obj.get("expires", 0)),
-            issued_at=int(obj.get("issued_at", 0)),
+            issued=int(obj.get("issued", 0)),
```

Comment:
Also are those fields even optional? If they are required and always returned by the server it's better to not have the default here at all.